src/encoded/regulome_search.py

```diff
@@ -413,8 +436,13 @@ def regulome_summary(context, request):
 
     # Redirect to regulome report for single unique region query
     if len(result['variants']) == 1:
-        query = {'regions': [v for v in result['variants']][0],
-                 'genome': result['assembly']}
+        query = {
+            'regions': [
+                '{}:{}-{}'.format(v['chrom'], v['start'], v['end'])
```

Comment:
why does the query expect this format?  I guess that's what the UI sends it.

Reply:
It can be anything which can be parsed properly here: https://github.com/ENCODE-DCC/regulome-encoded/blob/dev/src/encoded/regulome_search.py#L258-L279 But I assume it has to be a string since it's redirected to the other endpoint. Maybe I'm wrong?